desktop-widgets/tab-widgets/TabDiveInformation.cpp

```diff
@@ -43,6 +75,22 @@ void TabDiveInformation::clear()
 	ui->airTemperatureText->clear();
 	ui->atmPressVal->clear();
 	ui->salinityText->clear();
+	ui->waterTypeText->clear();
+}
+
+void TabDiveInformation::divesEdited(int i)
```

Comment:
Ultimately, this should be moved up one level and not reimplemented for every tab. I will look into it.

Reply:
This is part of the follow-up.